src/app/content/components/AssignedTopBar.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled, { css } from 'styled-components';
+import theme from '../../theme';
+import { setTextSize } from '../actions';
+import * as selectContent from '../selectors';
+import { LinkedArchiveTreeSection } from '../types';
+import { shadow, TopBarWrapper } from './Topbar/styled';
+import { TextResizer } from './Topbar/TextResizer';
+
+// tslint:disable-next-line:variable-name
+const StyledTopBarWrapper = styled(TopBarWrapper)`
+  ${shadow}
+
+  && {
+    top: 0;
+  }
+
+  background: #fff;
```

Comment:
Isn't this white in the regular view too?

Reply:
It is, updated to use the same theme constant for white.